### PR TITLE
Sign in using a email link

### PR DESCRIPTION
**Pull request by tatyree**
Closes https://trello.com/c/tle7jWgE/280-allow-users-to-access-their-accounts-via-magic-link

This PR does a bit of cleanup on the `apply` SPA Vue app and adds in basic authentication using an email link.  It does not yet use notify, nor does it do anything much beyond allowing the sign-in.

### CODE REVIEW

File: .env.staging [hunk 1]
```diff
@@ -0,0 +1,8 @@
+NODE_ENV=production
```
Comment (alexbokii) [inline .env.staging]: Is the NODE_ENV="production" same for both production and staging envs?

**Comment by tatyree** [comment]
> @tatyree I can see the `--token=$FIREBASE_DEPLOY_TOKEN` argument has been removed from the `firebase deploy` command. How/why does this work without it?

It's in `.circle/config.yml` now.

File: .env.staging (hunk 1 above)
Comment (tatyree) [reply .env.staging]: It is.  Staging runs in `production` (i.e. built) mode.

Comment (alexbokii) [reply .env.staging]: 👍

**Pull request merged**